src/docs/spell.ok

```diff
@@ -600,7 +613,9 @@ recnum
 recoverability
 recs
 rectype
+rel
```

Comment:
Sorry, missed this in the prior convo about `ful`. Can we use `release` in the labels and not whitelist this?

Reply:
acq-rel are fairly standard shortenings does it really matter that we've added a spelling to this arbitrary list?

Reply:
Yeah fair. Happy to leave this as is.